resources/generate_book.py

```diff
@@ -499,6 +499,18 @@ def make_book(rm: ResourceManager, i18n: I18n, local_instance: bool = False):
                 multiblock('', '', False, (('   ', '   ', '   '), ('   ', ' R ', '   '), ('   ', ' 0 ', '   ')), {'0': 'AIR', 'R': 'tfc:rock/raw/gabbro'}),
             ),
         )),
+        entry("leather_making","Leather Making", "minecraft:leather", pages=(
+            text("$(thing)Leather$() is a useful item that could be used in the creation of leather armor or used in crafting recipes, but it takes some steps to make it.$(br2)To get started you will first need to obtain hide. This can be done by slaughtering $(l:the_world/wild_animals)animals$().$(br2)Each $(l:the_world/wild_animals)animal$() can drop small, medium or large hide depending on it's size."),
```

Comment:
* Don't say 'a useful item'. Every item is useful.
* Don't say 'used in recipes', that's obvious
* Don't say 'but it takes steps to make it', just explain the steps
* Use small quotes (') not big quotes (")